tensorflow_lite_support/python/task/processor/proto/BUILD

```diff
@@ -1,4 +1,4 @@
-# Placeholder for internal Python strict library compatibility macro.
```

Comment:
Please don't delete this line as it will get replaced automatically in internal build.

Reply:
@khanhlvg Alright, will re-add it. However, I noticed it was removed in another commit sometime ago so decided to do the same here.

Reply:
@khanhlvg Fixed it.